python/cmake/FindPETSc4py.cmake

```diff
@@ -45,6 +45,16 @@ if(PETSC4PY_INCLUDE_DIRS)
   set(PETSC4PY_FIND_QUIETLY TRUE)
 endif(PETSC4PY_INCLUDE_DIRS)
 
+message(STATUS ${PYTHON_EXECUTABLE})
+
+execute_process(
```

Comment:
... and also this one: I cannot see any difference compared to lines 58-64

Reply:
Same as above, I had to debug the cmake because for some reason my cmake (started from my IDE) did not choose the right python executable. I'll revert these changes.